docs/legacy/introduction.md

```
@@ -124,3 +127,13 @@ Grease embraces this use case and optimizes the design and UX based on the follo
   claim their funds after a predetermined time-out. In this case, the
   forcing party is usually the merchant since they have the greater incentive to do so in the case where a channel
   has been abandoned by the client.
+
+### Anti-principles
+
+The following design goals are explicitly _excluded_ from the Grease design:
+
+* Multi-hop channels. Multi-hop channels are probably _possible_ in Grease, but they are not a design goal. 
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)